docs/reana.md

```diff
@@ -5,6 +5,30 @@ Explore Reana on the software's [official documentation](https://docs.reana.io/)
 
 You can find other examples of differen workflow languages on the [official Reana documentation](https://docs.reana.io/advanced-usage/access-control/rucio/).
 
+# Authentication with reana-client
+
+The REANA client provides a command-line interface for authenticating with the REANA server. You can use the `reana-client auth` command which initiates an OAuth 2.0 device flow authentication:
```

Comment:
Please add a link for oauth2.0 so that people can educate themselves, for example
https://auth0.com/intro-to-iam/what-is-oauth-2
or https://oauth.net/2/ 
(I prefer first link, more user-friendly. Thoughts @Soap2G)